src/PrizePool.sol

```diff
@@ -21,19 +21,19 @@ error DrawManagerAlreadySet();
 /// @notice Emitted when someone tries to claim a prize that was already claimed
 /// @param winner The winner of the prize
 /// @param tier The prize tier
-error AlreadyClaimedPrize(address winner, uint8 tier);
+error AlreadyClaimedPrize(address vault, address winner, uint8 tier, uint32 prizeIndex);
```

Comment:
We probably want the recipient in all of these events as well

Reply:
Yes; will add